.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+vs.code
+test.md
```

Comment:
-  [ ] You may have added test.md to gitignore but if you look at your code on GitHub, the test.md file still appears as seen in the screenshot below 👇 
![image](https://user-images.githubusercontent.com/45460192/170360887-0804212b-d965-480f-be42-d8400c6bcdf3.png)

Reply:
now i fix it please review it again thank you

Reply:
- Hello @basitali111 , please ask for another review on the student dashboard